### ISSUE TITLE
Add SpAero chemistry scheme

### ISSUE THREAD

**Issue by gold2718** [NorESMhub/CAM#103]
### What is the feature/what would you like to discuss?

Add the SpAero (specified aerosol) chemistry scheme to the NorESM2.3 release.

### Will this change (regression test) answers?

No

### Will you be implementing this enhancement yourself?

No

**Comment by gold2718** [comment]
@oyvindseland, @DirkOlivie, is this a duplicate of #12?

**Comment by oyvindseland** [comment]
Yes this is a duplicate of #12. Suggestion: Write in #12. Implementation moved to NorESM2.3, point to this issue and then close #12

**Comment by gold2718** [comment]
@DirkOlivie, is this still on the agenda for NorESM3.0? For NorESM3.5?

**Comment by oyvindseland** [comment]
I suggest to move it to 3.5 for now.
